server/tests/client/decentralized.spec.ts

```diff
@@ -61,5 +62,5 @@ function test (
   })
 }
 
-test('cleartext', clients.decentralized.ClearText)
-test('secure', clients.decentralized.SecAgg)
+test('cleartext', clients.decentralized.Base)
+test('secure', clients.decentralized.Base)
```

Comment:
do both tests still work if called on the same?

Reply:
I should actually delete the 'secure' one for the time being. As @swaglid explained to us, the current sec_agg is not working properly.  My goal is to get this PR merged quick enough for him to begin re-implementing secure aggregation.